packages/jest-emotion/test/react-enzyme.test.js

```diff
@@ -4,31 +4,22 @@ import * as enzyme from 'enzyme'
 import { jsx } from '@emotion/core'
 import { createSerializer as createEnzymeSerializer } from 'enzyme-to-json'
 import { createSerializer } from 'jest-emotion'
-import { toMatchSnapshot } from 'jest-snapshot'
 import React from 'react'
-
-const createEnzymeSnapshotMatcher = serializerOptions => {
-  const serializer = createEnzymeSerializer(serializerOptions)
-  const identityPrinter = v => v
-
-  return function(val) {
-    return toMatchSnapshot.call(this, serializer.print(val, identityPrinter))
-  }
-}
+import toJson from 'enzyme-to-json'
 
 expect.addSnapshotSerializer(createSerializer())
+expect.addSnapshotSerializer(createEnzymeSerializer())
 
-expect.extend({
-  toMatchShallowSnapshot: createEnzymeSnapshotMatcher(),
-  toMatchDeepSnapshot: createEnzymeSnapshotMatcher({ mode: 'deep' })
-})
+const expectToMatchSnapshot = component => {
```

Comment:
for cases like this we use `jest-in-case` helper

Reply:
Cool, will check it out.

Reply:
Looks similar to [`test.each`](https://jestjs.io/docs/en/api#testeachtablename-fn-timeout). I've refactored to use `jest-in-case`, let me know what you think.